java/core/src/main/java/co/worklytics/psoxy/rules/RuleSet.java

```diff
@@ -0,0 +1,7 @@
+package co.worklytics.psoxy.rules;
+
+public interface RuleSet {
+
+    @Deprecated // migrate to use transform-level ids
+    String getDefaultScopeIdForSource();
```

Comment:
thought is to define `scope` on the transform itself, as we have cases that mix scopes other than email (specifically the directory integrations have phone numbers, screen names, employee ids, etc)